src/mixins/filing-template-mixin.ts

```diff
@@ -593,6 +596,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
 
     // Store share classes and original resolution dates
     this.setShareClasses(cloneDeep(businessSnapshot.shareStructure.shareClasses))
+    this.setResolutionDates([]) // clear new resolution dates
```

Comment:
Bugfix.

Reply:
Can you elaborate on this?

Reply:
If the user clicked "delete" or "delete all" then any change to the New Resolution Dates must be cleared.

Reply:
ah ok :) I was overthinking this.